keras/backend/numpy_backend.py

```diff
@@ -257,11 +257,12 @@ def in_test_phase(x, alt, training=None):
 
 
 def relu(x, alpha=0., max_value=None, threshold=0.):
-    y = x * (x >= threshold)
```

Comment:
There is no answer in naming conventions. However, simple naming such as `x`, `y` is enough for readability and good for consistency in the NumPy backend.